projects/origin-release/ipi_deprovision/ipi-deprovision.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
```

Comment:
This file should be somewhere under `core-services`, or even not needed if we build the image with `ci-operator`.

Reply:
https://github.com/openshift/ci-tools/pull/26

Reply:
https://github.com/openshift/ci-tools/pull/26